.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - feat/dev-245-build-binaries
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+        python:
+          - 3.5
+          - 3.6
+          - 3.7
```

Comment:
Probably just pick one version instead

Reply:
I agree. 3.7 is probably a safe choice. There was some issue with wheels not being made for some of our pinned dependencies for py3.8 on Windows. It might go away if we upgrade the dependencies, assuming the newer deps have wheels built for them.